libs/wingsdk/src/cloud/secret.ts

```diff
@@ -22,6 +22,11 @@ export interface SecretProps {
    * @default - a new secret is provisioned with a generated name
    */
   readonly name?: string;
+
+  /**
+   * The secret's value.
+   */
+  readonly value: string;
```

Comment:
No... we don't allow secret in the code.

Reply:
Ok, I thought that the current implementation was a temporary workaround; I didn't understand that it was intentional and that Wing shouldn't allow you to handle secrets in the code.

Reply:
Btw, if we decide that an user can only import existing secrets, then the `name` argument will be mandatory to specify which one you want (currently it is optional and a new secret is created if a name isn't provided).

@eladb Are you fine with this change?